src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -232,3 +234,6 @@ class DepositServlet(app: EasyDepositApiApp)
     }
   }
 }
+object DepositServlet {
+  val uploadRoot = "original"
```

Comment:
is there a better place for this constant?

Reply:
see commit comments